src/coffee/commands/images-convert.coffee

```diff
@@ -12,7 +12,7 @@ program
 .option '-d, --descriptions <path>', 'set image descriptions file path'
 .option '-r, --regex [name]', 'an optional RegExp used for filtering listed files (e.g.: /(.*)\.jpg/)', ''
 .option '-l, --logFile <path>', 'optionally log to a file instead of printing to console (errors will still be printed to stderr)'
-.option '-ic, --compress', 'optionally compress images', false
+.option '-C, --compress', 'optionally compress images', false
```

Comment:
`-c` was taken by `--credentials`, and more than 1 letter for short flag is considered as multiple short flags (eg. `-ic` = `-i -c`). I, wanted to use first letter of a command and other synonimous weren't proper (like minify).